modules/adb-exfiltration-protection/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "workspace_id" {
+  // The ID of the Databricks Workspace in the Azure management plane.
+  value = azurerm_databricks_workspace.this.id
+}
+
+output "workspace_url" {
+  // The workspace URL which is of the format 'adb-{workspaceId}.{random}.azuredatabricks.net'
+  // this is not named as DATABRICKS_HOST, because it affect authentication
+  value = "https://${azurerm_databricks_workspace.this.workspace_url}/"
+}
+
+output "rg_id" {
```

Comment:
`rg_id` - let make it `resource_group_id` instrad

Reply:
I actually changed it to `azure_resource_group_id`